src/Commands/ProjectCreateCommand.php

```diff
@@ -139,6 +140,66 @@ public function validateCreateProject(CommandData $commandData)
         $this->providerManager()->validateCredentials();
     }
 
+    /**
+     * Get CI short name based on the CI provider.
+     */
+    public function getCiShortName($ci_provider) {
+        $reflect = new \ReflectionClass($ci_provider);
+        switch ($reflect->getShortName()) {
+            case 'CircleCIProvider':
+                return 'circleci';
+
+            case 'GitLabCIProvider':
+                return 'gitlabci';
+
+            case 'BitbucketPipelinesProvider':
+                return 'bitbucket';
+
+            case 'GithubActionsProvider':
+                return 'github';
+        }
+    }
+
+    /**
+     * Copy CI files from the given/default repo.
+     */
+    public function copyCiFiles($ci_provider, $created_folder, $cms_version = 'd8', $repo = '') {
```

Comment:
Pending: Add ci-repo option. @greg-1-anderson could you confirm that 'ci-repo' is a good name for the repo to use for the CI integrations?

Reply:
How about `--ci-template`?

Reply:
Done. Thanks